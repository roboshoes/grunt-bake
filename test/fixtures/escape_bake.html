<!DOCTYPE html>
<html>
    <head></head>
    <body>
        <!--(bake-escape includes/unescaped.html)-->
    </body>
</html>