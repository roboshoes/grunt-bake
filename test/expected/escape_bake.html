<!DOCTYPE html>
<html>
    <head></head>
    <body>
        new ArrayList&lt;Integer&gt;() {{
            add(1);
            add(2);
        }};
    </body>
</html>